src/xcode/ENA/ENAUITests/ExposureSubmission/ENAUITests_04a_ExposureSubmission.swift

```diff
@@ -475,6 +475,7 @@ class ENAUITests_04a_ExposureSubmission: CWATestCase {
 	
 	func test_exposureSubmissionSuccess_screen() {
 		launchAndNavigateToSymptomsScreen()
+		app.buttons[AccessibilityIdentifiers.ExposureSubmission.primaryButton].waitAndTap()
```

Comment:
Should this be part of the `launchAndNavigateToSymptomsScreen()` function. Does it not lead all the way to the symptoms screen anymore?